src/redux/reducers/notificationReducer.ts

```diff
@@ -1,48 +1,13 @@
-import { v4 as uuidv4 } from 'uuid';
-
-import i18n from '../../translations/i18n';
 import { Notification } from '../../types/notification.types';
-import { AuthActions, LOGIN_FAILURE, REFRESH_TOKEN_FAILURE } from '../actions/authActions';
-import {
-  ADD_NOTIFICATION,
-  CLEAR_NOTIFICATIONS,
-  NotificationActions,
-  REMOVE_NOTIFICATION,
-} from '../actions/notificationActions';
-import { ORCID_REQUEST_FAILURE, OrcidActions } from '../actions/orcidActions';
-import { SEARCH_FAILURE, SearchActions } from '../actions/searchActions';
-import { SET_USER_FAILURE, SET_USER_SUCCESS, UserActions } from '../actions/userActions';
+import { SET_NOTIFICATION, NotificationActions, REMOVE_NOTIFICATION } from '../actions/notificationActions';
 
-export const notificationReducer = (
-  state: Notification[] = [],
-  action: NotificationActions | AuthActions | OrcidActions | UserActions | NotificationActions | SearchActions
-) => {
+export const notificationReducer = (state: Notification | null = null, action: NotificationActions) => {
   switch (action.type) {
-    case ORCID_REQUEST_FAILURE:
-    case REFRESH_TOKEN_FAILURE:
-    case SET_USER_FAILURE:
-    case SEARCH_FAILURE:
-    case LOGIN_FAILURE:
-      return [...state, { key: uuidv4(), ...action }];
-    case ADD_NOTIFICATION:
-      return [...state, { ...action.notification }];
+    case SET_NOTIFICATION:
+      return action.notification;
```

Comment:
er det bare en notification som vises om gangen nå?

Reply:
hva om flere ting går galt? skal vi ikke vise flere feilmeldinger?

Reply:
bare en om gangen nå ja. vi fant ut at det kunne være like greit å gjøre det enklest mulig siden dette uansett bare er noe vi har funnet ut at trengs, og det ikke er noen bestilling/spec på dette.
Material fraråder faktisk stacking av beskjeder også: "It also enables you to stack them on top of one another (although this is discouraged by the Material Design specification)" https://material-ui.com/components/snackbars/#notistack

Når det er sagt så skal det være en relativt smal sak å legge til støtte for flere senere om vi gjør dette til en liste i stedet

Reply:
okidok 👍